bin/templates/cordova/lib/builders/GradleBuilder.js

```diff
@@ -211,3 +209,46 @@ module.exports = GradleBuilder;
 function isAutoGenerated(file) {
     return fs.existsSync(file) && fs.readFileSync(file, 'utf8').indexOf(MARKER) > 0;
 }
+
+/**
+ * A special superspawn-like implementation, required to workaround the issue
```

Comment:
It looks like `superspawn` should be modified to take the same parameters as `child_process.spawn`. Refer [here](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) for Node 5.0.x, and [here](https://nodejs.org/docs/latest-v0.12.x/api/child_process.html#child_process_child_process_spawn_command_args_options) for Node 0.12.x.